tests/e2e/e2e.go

```diff
@@ -363,143 +316,101 @@ func (test *TestCtx) VppRelease() string {
 }
 
 // GenericClient provides generic client for communication with default VPP-Agent test component
+// Deprecated: use ctx.Agent.GenericClient(...) instead
 func (test *TestCtx) GenericClient() client.GenericClient {
-	c, err := test.agentClient.GenericClient()
-	if err != nil {
-		test.t.Fatalf("Failed to get generic VPP-agent client: %v", err)
-	}
-	return c
+	test.t.Helper()
+	return test.Agent.GenericClient()
 }
 
 // GRPCConn provides GRPC client connection for communication with default VPP-Agent test component
+// Deprecated: use ctx.Agent.GRPCConn(...) instead
```

Comment:
The previous comment applies to this comment as well.. and the others that basically provide quick access to methods of the main agent.